### DISCUSSION TITLE
Feed of Releases & repo arhivation based on Stars

### DISCUSSION THREAD

**Discussion by QuAzI** [community/community#49140] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

It would be nice to have a feed with new releases for repositories which marked by user with Star. And optionally for repositories where user made some pull-reuests, comments.

**Comment by BellaBijl** [comment]
> ### Select Topic Area
> General
> 
> ### Body
> It would be nice to have a feed with new releases for repositories which marked by user with Star. And optionally for repositories where user made some pull-reuests, comments.

**Comment by BellaBijl** [comment]
I’m just saying if you want me too I’m going to

**Comment by BellaBijl** [comment]
The most recent study by researchers from University Hospital London showed the same results for the average American

**Comment by ApichaiBaokeaw** [comment]
- [ ]

**Reply by ApichaiBaokeaw** [reply]
Hi

**Reply by Devineonline99** [reply]
Hello

**Comment by Emadalshamery** [comment]
Gooooo

**Reply by Minicoolboy77777** [reply]
Tnx

**Comment by Emadalshamery** [comment]
> ### Select Topic Area
> 
> General
> 
> ### Body
> 
> It would be nice to have a feed with new releases for repositories which marked by user with Star. And optionally for repositories where user made some pull-reuests, comments.

**Comment by Emadalshamery** [comment]
Alemadtr.com@gmail.com

**Comment by Emadalshamery** [comment]
Alemadtr.com@gmail.com

**Comment by Amg845** [comment]
https://github.com/orgs/community/discussions/49140

**Reply by Amirkahanias** [reply]
A.kahani21s

**Comment by Sheena131** [comment]
https://www.facebook.com/blue1519

**Comment by Azem92** [comment]
- [ ]

**Comment by F8ejhdjijfh** [comment]
> https://github.com/orgs/community/discussions/49140

**Comment by M-s-s-a** [comment]
> Hello

**Comment by Al3676** [comment]
Excellent

**Comment by hyp773** [comment]
Help me get the new phone may

**Reply by zaaki555** [reply]
What a phone

**Comment by LashCo** [comment]
Response #1

**Comment by Al3676** [comment]
Thanksgiving services

**Reply by zaaki555** [reply]
Thank you

**Reply by zaaki555** [reply]
Thank you

**Reply by samsul79** [reply]
💪👍

**Reply by Brightomeham4** [reply]
I really need support

**Comment by zaaki555** [comment]
Thank you

**Reply by DastagirHossion** [reply]
All done

**Reply by DastagirHossion** [reply]
> All done 

Product feedback

**Reply by thanhtai3979** [reply]
look good to me

**Comment by MaxeymichaelGracechristilynn** [comment]
> ### Select Topic Area
> 
> General
> 
> ### Body
> 
> It would be nice to have a feed with new releases for repositories [which marked]() by user with Star. And optionally for repositories where user made some pull-reuests, comments.

**Comment by Koleskiz** [comment]
Hi

**Comment by genesis-solution** [comment]
Hi

**Comment by krud491** [comment]
To receive updates on new releases for repositories you've starred and optionally for repositories where you've contributed with pull requests or comments, you can leverage GitHub's notification settings. By configuring your notification preferences, you can choose to receive notifications for new releases and activity on repositories you're interested in. Additionally, you can explore third-party tools or services that offer more customized notification features based on your GitHub activity.

**Reply by krud491** [reply]
It sounds like you're interested in setting up a way to get notifications for new releases from GitHub repositories you've interacted with, either by starring them or contributing through pull requests and comments. Here's a step-by-step guide on how to do this:

1. **Watch Repositories**: For repositories you're interested in, you can click on the 'Watch' button at the top of the repository page on GitHub. Set it to "Releases only" to get notifications for new releases.

2. **GitHub Notifications**: Make sure your GitHub notifications are set up to alert you in the way you prefer (email, web, or mobile). You can customize this in your GitHub settings under 'Notifications'.

**Reply by krud491** [reply]
**Starred Repositories Feed**:
   - You can use the GitHub API to create a custom feed. This involves using the GitHub API endpoint for starred repositories (`/users/:username/starred`) and the releases endpoint (`/repos/:owner/:repo/releases`) to pull the latest release information.
   - Tools like RSS or Atom feeds can be used to keep track of new releases if you prefer not to build your own tool. Services like RSSHub can create feeds from your starred repositories.

4. **Enhanced Notifications with Third-Party Tools**: Consider third-party services like Sibbell or Release Bell, which can notify you of new releases for repositories you've starred on GitHub.

5. **Browser Extensions**: Browser extensions such as OctoLinker or GitHunt can also notify you about new releases, among other features that enhance your GitHub browsing experience.

6. **Automate with GitHub Actions**: If you're comfortable with GitHub Actions, you can set up a workflow that checks for new releases and sends you a notification through email, Slack, or other methods.

**Comment by Alexander8186** [comment]
```
- > ``
```

**Comment by Alexander8186** [comment]
****

**Comment by Abirhasankas143** [comment]
cc92ff10158115ef8abc16d1b4c2b1ee380b98ea

**Comment by Abirhasankas143** [comment]
#eae03dca69e5e4a8c52acf162fb5a07243a57709

**Comment by krud491** [comment]
Okay there's a reply and as a matter of fact I replied last time my
response was up there you don't have to send me these nasty messages bro

**Comment by Al3676** [comment]
Exxelent

**Comment by Somaticbeing84** [comment]
> ### Select Topic Area
> 
> General
> 
> ### Body
> 
> It would be nice to have a feed with new releases for repositories which marked by user with Star. And optionally for repositories where user made some pull-reuests, comments.

**Comment by genesis-solution** [comment]
ok

**Comment by Asilkan5807** [comment]
- [ ]

**Reply by Asilkan5807** [reply]
- [ ]

**Comment by zaaki555** [comment]
What would you like me to answer and why Answer!

ned, 26. maj 2024. 19:38 pošiljalac samsul79 ***@***.***> je
napisao:

**Reply by Szymon580** [reply]
4sJ2gQAnkShZrEP

**Comment by Khaledtbs** [comment]
> https://github.com/orgs/community/discussions/49140

**Comment by Nurtrisariyanto** [comment]
Okey

**Comment by Nurtrisariyanto** [comment]
👍👍👍👍

**Comment by Nurtrisariyanto** [comment]
Usum telo

**Comment by Nguyenhung078** [comment]
> > ### Select Topic Area
> > General
> > 
> > ### Body
> > It would be nice to have a feed with new releases for repositories which marked by user with Star. And optionally for repositories where user made some pull-reuests, comments.
> 
>

**Comment by Aminzarbani** [comment]
X-GitHub-Api-Ver
